src/sage/algebras/finite_dimensional_algebras/finite_dimensional_algebra_ideal.py

```diff
@@ -39,17 +41,21 @@ class FiniteDimensionalAlgebraIdeal(Ideal_generic):
 
     EXAMPLES::
 
+        sage: cat = CommutativeAlgebras(GF(3)).FiniteDimensional().WithBasis()
         sage: A = FiniteDimensionalAlgebra(GF(3), [Matrix([[1, 0], [0, 1]]),
-        ....:                                      Matrix([[0, 1], [0, 0]])])
+        ....:                                      Matrix([[0, 1], [0, 0]])],
+        ....:     category=cat)
```

Comment:
```suggestion
        ....:                                      Matrix([[0, 1], [0, 0]])],
        ....:                               category=cat)
```
I think this alignment is more in the spirit of PEP8?